examples/ex04/ex04b/Displacement2.csv

```diff
@@ -0,0 +1,500 @@
+3.01,0.0002

```

Comment:
If this file is same as the one in 4a or one of the other examples, you can provide the file path in the functions block. For example if the file is in example 4a, then in functions block data_file = '../ex04a/Displacement2.csv'. Once you do that, please remove this file from your commit using git rm file_name.